build-interactive-menus.md

```diff
@@ -10,6 +10,10 @@ Using the `app:install` Artisan command you can install the `menu` component:
 php <your-app-name> app:install menu
 ```
 
+**Note:** This component only works on systems with the php extension
+`ext-posix` enabled. **This will prevent your application to work on
+Windows as `ext-posix` is not available on Windows systems**
```

Comment:
I'd probably add this as a quote block so that it stands out more, but otherwise looks good!

```suggestion
> **Note:** This component only works on systems with the php extension
`ext-posix` enabled. **This will prevent your application to work on
Windows as `ext-posix` is not available on Windows systems**
```